src/logic/operate.js

```diff
@@ -0,0 +1,15 @@
+import Big from 'big.js';
+
+Big.strict = true;
+
+const operate = (numberOne, numberTwo, operation) => {
+  const BigMethods = {
+    '-': Big(numberOne).minus(Big(numberTwo)),
+    '+': Big(numberOne).plus(Big(numberTwo)),
+    '÷': Big(numberOne).div(Big(numberTwo)),
+    x: Big(numberOne).times(Big(numberTwo)),
```

Comment:
### [Optional]
Let's practice DRY code, instead of repeating yourself `Big`, you could do it this way;
`let numberOne = Big(firstNumber)`